packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -689,18 +706,35 @@ export const makeInnerVault = (
     );
     manager.reallocateReward(fee, vaultSeat, seat);
 
+    // @ts-expect-error proposals not generic (RUN implies NatValue)
     refreshLoanTracking(oldDebt, oldCollateral, stagedDebt);
 
-    return setupOuter(innerVault);
+    const { transferInvitationHook, updater } = setupOuter(innerVault);
+    outerUpdater = updater;
+    updateUiState();
+
+    return transferInvitationHook;
   };
 
+  /**
+   *
+   * @param {ZCFSeat} seat
+   * @returns {import('./vaultKit.js').TransferInvitationHook}
+   */
   const makeTransferInvitationHook = seat => {
     assertVaultIsOpen();
     seat.exit();
+
     // eslint-disable-next-line no-use-before-define
-    return setupOuter(innerVault);
+    const { transferInvitationHook, updater } = setupOuter(innerVault);
+    outerUpdater = updater;
+    updateUiState();
+
+    return transferInvitationHook;
```

Comment:
This is named wrong.  This method is "(makeTransferInvitation)Hook", not "make(TransferInvitationHook)"